### ISSUE TITLE
Accordion widget does not work

### ISSUE THREAD

**Issue by DougRzz** [pbugnion/ipywidgets_server#16]
The accordion widget doesn't collapse down.   The example below works in Jupyter but not in ipywidgets-server.  

```python
import ipywidgets as widgets

accordion = widgets.Accordion(children=[widgets.IntSlider(), widgets.Text()])
accordion.set_title(0, 'Slider')
accordion.set_title(1, 'Text')
accordion
vbox = widgets.VBox([accordion])
```

**Comment by pbugnion** [comment]
Thanks for testing this. 

That's funny -- it works fine here. What browser (and what version) are you using?

**Comment by DougRzz** [comment]
Latest Chrome on Windows 10.  I will download Firefox and test it on that.

![accordion](https://user-images.githubusercontent.com/3510318/33376828-24547132-d507-11e7-8dd9-8bd0fefe61b1.gif)

**Comment by DougRzz** [comment]
Firefox also has the same issue.   Interestingly, Firefox is nearly twice as quick to reload this ipywidget-server session.  Firefox = 8s. Chrome = 15s.

**Comment by pbugnion** [comment]
Very useful. I clearly missed some CSS magic that ipywidgets is doing to make this work cross-platform. I'll try to get my hands on a Windows VM to test this out.

Thanks for reporting.

**Comment by DougRzz** [comment]
@pbugnion 
Hi Pascal,  I thought I would try and investigate this accordion issue on windows.  I don't know a lot about web technologies such as javascript and css.... so I didn't get very far. 

I inspected the accordion widgets in jupyter and from the ipywidgets-server version using Chrome 'inspect element'.  I noticed that there was a difference in the  p.Collapse style.  There doesn't seem to be the webkit styling properties in the ipywidgets-server option.  I looked up https://webkit.org/  and it doesn't appear to be a web browser engine used by Windows. I am probably completely on the wrong track, do you have any thoughts on how I might get to the bottom of this?

Jupyter p.Collapse style:
![image](https://user-images.githubusercontent.com/3510318/37244667-4d8a27ee-2495-11e8-9b04-1774db5cc8bd.png)

Ipywidgets-server p.Collapse style:
![image](https://user-images.githubusercontent.com/3510318/37244677-60ef53a4-2495-11e8-845d-e80397f822db.png)

**Comment by pbugnion** [comment]
Ah that's really interesting. Thanks! I think you're exactly right -- those additional properties present in the stylesheet you see in the notebook provide browser compatibility.

*ipywidgets-server* just uses the same stylesheet as ipywidgets. The [ipywidgets stylesheet](https://github.com/jupyter-widgets/ipywidgets/blob/master/packages/controls/css/widgets-base.css#L995) defines `.p-Collapse` as just:

```css
.p-Collapse {
    display: flex;
    flex-direction: column;
    align-items: stretch;
}
```

One would therefore expect to see those properties present in both the notebook and ipywidgets-server. The issue seems to be that ipywidgets also passes its stylesheet through cssnext [here](https://github.com/jupyter-widgets/ipywidgets/blob/3af1317a0b6a9626cefb50b6a855fa9114ee54c8/widgetsnbextension/webpack.config.js#L20). It looks like cssnext sees the flex display and adds a bunch of browser compatibility features -- you can try it out [here](http://cssnext.io/playground/).

The right solution would be to add a cssnext preprocessor in `webpack.config.js`. This should create a stylesheet that's identical to the ipywidgets one.

I'm happy to make that change and release an alpha release that you can test out, unless you'd prefer to submit a PR? You've done all the hard work on this.

**Comment by DougRzz** [comment]
@pbugnion i partially understand. It sounds promising. I would be very happy to try an alpha if you have the time to implement. I wouldn't say i did most of the work. I was just floundering. A fix for this issue would be v nice for my dashboard.

**Comment by pbugnion** [comment]
Fixed by PR #32 .
